service/rpc/rpc.proto

```diff
@@ -56,61 +57,6 @@ service GnomobileService {
   rpc Hello(HelloRequest) returns (HelloResponse);
 }
 
-enum KeyType {
-  TypeLocal = 0;
-  TypeLedger = 1;
-  TypeOffline = 2;
-  TypeMulti = 3;
-}
-
-message KeyInfo {
-  KeyType type = 1;
-  string name = 2;
-  bytes pub_key = 3;
-  bytes address = 4;
-}
-
-message ListKeyInfoRequest {}
-
-message ListKeyInfoResponse {
-  repeated KeyInfo keys = 1;
-}
-
-message CreateAccountRequest {
-  string name_or_bech32 = 1;
-  string mnemonic = 2;
-  string bip39_passwd = 3;
-  string password = 4;
-  uint32 account = 5;
-  uint32 index = 6;
-}
-
-message CreateAccountResponse {
-  KeyInfo key = 1;
-}
-
-message SelectAccountRequest {
-  string name_or_bech32 = 1;
-}
-
-message SelectAccountResponse {
-  KeyInfo key = 1;
-}
-
-message GetActiveAccountRequest {}
-
-message GetActiveAccountResponse {
-  KeyInfo key = 1;
-}
-
-message HelloRequest {
-  string name = 1;
-}
-
-message HelloResponse {
-  string greeting = 1;
-}
-
 enum ErrCode {
```

Comment:
We still use ErrCode as an `Enum` because converting it into a `int` would be a nightmare for devs.

Reply:
I agree. The ErrCode is only in the gRPC interface. Do you want me to add a comment in the code?

Reply:
Yes we can explain why we keep it here and put the link to this PR too. Thanks

Reply:
I added comment "The ErrCode enum defines errors for gRPC API functions. These are converted from the Go error types returned by gnoclient."